grafana/grafana.libsonnet

```diff
@@ -137,6 +139,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
       local c =
         container.new('grafana', $._config.imageRepos.grafana + ':' + $._config.versions.grafana) +
         (if std.length($._config.grafana.plugins) == 0 then {} else container.withEnv([env.new('GF_INSTALL_PLUGINS', std.join(',', $._config.grafana.plugins))])) +
+        container.withEnvMixin([env.new('GF_SERVER_HTTP_PORT', '' + $._config.grafana.containerPort.port)]) +
```

Comment:
I think this can be done via the configuration file as well no? I think I'd prefer to keep all configuration possible within the config file. `GF_INSTALL_PLUGINS` is only there because it's the only way to configure plugins to install.